spec/requests/queries_spec.rb

```diff
@@ -62,12 +62,13 @@ def create_vms(count)
       expect_single_resource_query("id" => vm1.id.to_s, "href" => api_vm_url(nil, vm1), "guid" => vm1.guid)
     end
 
-    it 'supports compressed ids' do
+    specify 'support for compressed ids has been deprecated' do
```

Comment:
I'm *super* impressed you changed it from `it` to `specify`/`example` 😍

Reply:
Ugh *ANOTHER* rspec syntax...please no :(

Reply:
TBH I actually didn't know that there was an alias as `specify` either lol. I usually use `example`...